### PR TITLE
[docs] Refactor Strapi

### PR DESCRIPTION
**Pull request by ritwickrajmakhal**
## Description

Refactor Strapi handler docs.

## Type of change

- [x] 📄 This change requires a documentation update

## Additional Media:
image

- [x] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by ritwickrajmakhal** [comment]
@martyna-mindsdb please review it

File: docs/integrations/app-integrations/strapi.mdx [hunk 1]
```diff
@@ -60,18 +46,6 @@ FROM myshop.<collection-name>
 WHERE <field-name> = <value>;
```
Comment (martyna-mindsdb) [inline docs/integrations/app-integrations/strapi.mdx:46]: Let's add a LIMIT clause after WHERE.

**Comment by martyna-mindsdb** [comment]
Thanks @ritwickrajmakhal 
One small update above.

**Review by martyna-mindsdb — APPROVED** [review]

**Comment by martyna-mindsdb** [comment]
@ritwickrajmakhal 
Please `merge staging` to this branch, so we can merge it.

**Comment by ritwickrajmakhal** [comment]
@martyna-mindsdb Done

**Review by martyna-mindsdb — APPROVED** [review]

**Pull request merged**